### PR TITLE
task(issue-179): sales backend populated in the tables with add sales button 

### PR DESCRIPTION
**Pull request by abirshah**

### CODE REVIEW

File: ERP/routes/web.php [hunk 1]
```diff
@@ -140,6 +140,13 @@
 Route::get('/accountant', [AccountantController::class, 'goToAccoutantView'])
     ->name('accountant');
 
+    // Executes "sales" method in the AccountantController when the route is "/sales".
+Route::get('/sales', [AccountantController::class, 'sales'])
+->name('sales');
+
+Route::post('/sales', [AccountantController::class, 'saveSaleOrder'])
+->name('save.sale.order');
+
```
Comment (CeliaCaii) [inline ERP/routes/web.php:149]: Use the SalesController

File: ERP/app/Http/Controllers/AccountantController.php [hunk 2]
```diff
@@ -2,9 +2,11 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/AccountantController.php:5]: Why are you removing this?

File: ERP/app/Http/Controllers/AccountantController.php [hunk 3]
```diff
@@ -22,4 +24,36 @@ public function goToAccoutantView()
...
+        if ($sale->save()) {
+            $bike_sale_pivot = [
+                "bike_id" => $body["bicycleId"], "sale_id" => $sale->id,
+                'quantity_sold' => $body["quantitySold"]
+            ];
+            $sale->bikes()->sync([1 => $bike_sale_pivot]);
+        }
+
+        return $this->sales();
+    }
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/AccountantController.php:58]: I think this should be in SalesController

**Review by CeliaCaii — CHANGES_REQUESTED** [review]

**Review by pirasunt — APPROVED** [review]
nice job

**Review by CeliaCaii — APPROVED** [review]

**Pull request merged**